litmus-portal/frontend/k8s-deployment.yml

```diff
@@ -0,0 +1,34 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: maya-frontend-deployment
```

Comment:
Need to rename from `maya` to a generic name in all the places

Reply:
Rename it to litmus-portal-ui

Reply:
done